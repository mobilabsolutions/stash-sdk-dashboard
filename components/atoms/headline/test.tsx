/* eslint-env jest */

import React from 'react'
import renderer from 'react-test-renderer'

import { withTestSetup } from '../../../test_utils'
import H1 from './h1'
import H2 from './h2'

const TestH1 = withTestSetup(H1)
const TestH2 = withTestSetup(H2)

it('H1 Should Render', () => {
  const component = renderer.create(<TestH1>Heading 1</TestH1>)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})

it('H2 Should Render', () => {
  const component = renderer.create(<TestH2>Heading 2</TestH2>)
  const tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
