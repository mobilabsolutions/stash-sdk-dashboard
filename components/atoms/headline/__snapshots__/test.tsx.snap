// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 Should Render 1`] = `
<h1
  className="h1-yzcs6s-0 dyoCbE"
>
  Heading 1
</h1>
`;

exports[`H2 Should Render 1`] = `
<h2
  className="h2-ewzs3l-0 BtsSc"
>
  Heading 2
</h2>
`;
